packages/webapp/src/components/Task/TaskAnimalInventory/index.jsx

```diff
@@ -0,0 +1,120 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import MultiStepPageTitle from '../../PageTitle/MultiStepPageTitle';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import Form from '../../Form';
+import { useForm } from 'react-hook-form';
+import Button from '../../Form/Button';
+import { Main } from '../../Typography';
+import AnimalInventory, { View } from '../../../containers/Animals/Inventory';
+import styles from './styles.module.scss';
+
+export default function PureTaskAnimalInventory({
+  onContinue,
+  onGoBack,
+  persistedFormData,
+  useHookFormPersist,
+  history,
+  isDesktop,
+}) {
+  const { t } = useTranslation();
+  const ANIMAL_IDS = 'animalIds';
+
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    setValue,
+    formState: { isValid },
+  } = useForm({
+    mode: 'onChange',
+    shouldUnregister: false,
+    defaultValues: {
+      ...persistedFormData,
+      [ANIMAL_IDS]: persistedFormData.animalIds ?? [],
```

Comment:
I seem to like commenting on `??` 😂 
Shouldn't this be `persistedFormData.animalIds || []`?
I don't think we need to allow `[ANIMAL_IDS]` to be null, false or ''.

Reply:
I seem to miss these errors! haha yah you are right I think